lib/process-services-cloud/src/lib/form/components/widgets/radio-buttons/radio-buttons-cloud.widget.scss

```diff
@@ -9,10 +9,16 @@
         margin-bottom: 15px;
         display: flex;
         flex-direction: row;
+        align-items: flex-start;
+
+        .adf-label {
+            width: auto;
```

Comment:
`auto` is the default value for `width` and `height`, do we need to set those rules?